include/jemalloc/internal/tcache_inlines.h

```diff
@@ -62,7 +114,8 @@ tcache_alloc_small(tsd_t *tsd, arena_t *arena, tcache_t *tcache,
 		if (unlikely(arena == NULL)) {
 			return NULL;
 		}
-		if (unlikely(tcache_small_bin_disabled(binind, bin))) {
+		if (unlikely(tcache_small_bin_disabled(binind, bin,
+		    tsd_thread_nhbins_get(tsd)))) {
```

Comment:
We should always do the `thread_nhbins_get` variant, right?

Reply:
When initializing, we should use `tsd_thread_nhbins_set` instead of `thread_nhbins_set`. Thus , symmetrically, I use `tsd_thread_nhbins_get` as well. Yet for get, either is fine. I can change to `thread_nhbins_get` if that fits the conventions better.

Reply:
Yeah during special cases like init, the tsd variant can be used to get or set unconditionally. But otherwise the vast majority of use cases should go through the wrappers (which includes the additional asserts).